table-api/src/main/java/io/deephaven/api/updateby/UpdateByOperation.java

```diff
@@ -1635,11 +1637,13 @@ static UpdateByOperation RollingStd(long revTicks, String... pairs) {
     }
 
     /**
-     * Create a {@link RollingStdSpec rolling standard deviation} for the supplied column name pairs, using ticks as the
-     * windowing unit. Ticks are row counts and you may specify the reverse and forward window in number of rows to
-     * include. The current row is considered to belong to the reverse window but not the forward window. Also, negative
-     * values are allowed and can be used to generate completely forward or completely reverse windows. Here are some
-     * examples of window values:
+     * Create a {@link RollingStdSpec rolling sample standard deviation} for the supplied column name pairs, using ticks
+     * as the windowing unit. Sample standard deviation is computed using Bessel's correction:
+     * https://en.wikipedia.org/wiki/Bessel%27s_correction This ensures that the sample variance will be an unbiased
+     * estimator of population variance. Ticks are row counts and you may specify the reverse and forward window in
+     * number of rows to include. The current row is considered to belong to the reverse window but not the forward
+     * window. Also, negative values are allowed and can be used to generate completely forward or completely reverse
+     * windows. Here are some examples of window values:
```

Comment:
In all of these places, the bessel part should be a separate paragraph after the original paragraph.